keps/sig-network/2091-admin-network-policy/README.md

```diff
@@ -105,7 +105,7 @@ a new API that captures the administrator's intent.
 
```

Comment:
Nit: above this on line we should change the "It doesn't include priorities, or any concept of ..." line to something like 

- The commutative nature of NetworkPolicy can make certain filtering intents difficult to express.

since NP was designed to be like this

Reply:
Done